src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import glamorous from 'glamorous';
+
+import Button from '../Button/Button';
+import constants from '../../constants';
+
+const { borderRadius } = constants;
+
+const ButtonGroupContainer = glamorous.div({
+   display: 'inline-block',
+});
+
+const GroupedButton = glamorous(Button)({
+   borderRadius: 0,
+
+   '&:first-of-type': {
+      borderTopLeftRadius: borderRadius,
+      borderBottomLeftRadius: borderRadius,
+   },
+   '&:last-of-type': {
+      borderTopRightRadius: borderRadius,
+      borderBottomRightRadius: borderRadius,
+   },
+});
+
+const ButtonGroup = ({
+   children,
+   className,
+   disabled,
+   onMouseEnter,
+   onMouseLeave,
+}) => (
+   <ButtonGroupContainer
+      className={className}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
```

Comment:
Why do we need explicitly pass `onMouseEnter` and `onMouseLeave` to the `ButtonGroupContainer`? I think a better approach might be to collect the unused props and spread them onto the container. Like this:

```jsx
const ButtonGroup = ({ children, disabled, ...props }) => (
  <Div display="inline-block" {...props}>
    ...
  </Div>
);
```

Notice that I used `Div` instead of `ButtonGroupContainer`. Since `ButtonGroupContainer` only contains the `display: inline-block` rule,  this is a good place to use glamorous' `Div` component and simply pass a `display` prop instead of defining a new component. From the glamorous docs:

> Having to name all of that stuff could be tedious, so having these pre-built components is handy. The other handy bit here is that the props are the styles of these components.

I like to use the built-in components for layout purposes like setting `display` or `position`.